pom.xml

```diff
@@ -1,18 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 org.fakekoji.http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <modules>
+        <module>jenkins-scm-koji-plugin</module>
+        <module>fake-koji</module>
+        <module>koji-scm-lib</module>
+    </modules>
     <parent>
```

Comment:
I don't like that this whole project has as parent jenkins plugin. It makes sense just for `jenkins-scm-koji-plugin`. Unfortunately I don't know ideal solution for that. Can you please take a look if you can come up with something?